packages/ui-library/src/elements/tag-input/index.js

```diff
@@ -38,16 +39,13 @@ const Tag = ( { tag, index, disabled = false, onRemoveTag, screenReaderRemoveTag
 
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
-		<div onKeyDown={ handleKeyDown } className="yst-tag-input__tag" { ...props }>
+		<Badge onKeyDown={ handleKeyDown } { ...props }>
```

Comment:
Let's use the `variant="plain"` here to keep the prevent the tag input from being to flashy

Reply:
Good point. Done.